src/nvim/os/os_defs.h

```diff
@@ -18,11 +18,15 @@
 #endif
 
 #if defined(UFS_MAXNAMLEN) && !defined(MAXNAMLEN)
-# define MAXNAMLEN UFS_MAXNAMLEN    /* for dynix/ptx */
+# define MAXNAMLEN UFS_MAXNAMLEN    // for dynix/ptx
```

Comment:
10.01.2016, 22:09, "Justin M. Keyes" <notifications@github.com>:
> In src/nvim/os/os_defs.h:
>
>> @@ -18,11 +18,15 @@ > #endif > > #if defined(UFS_MAXNAMLEN) && !defined(MAXNAMLEN) > -# define MAXNAMLEN UFS_MAXNAMLEN /* for dynix/ptx */ > +# define MAXNAMLEN UFS_MAXNAMLEN // for dynix/ptx
>
> Must use /* comments inside macros (and lint should allow this, or ignore lint in that case).
>
> Note that inline comments are still preferred for concise descriptions like this, so I do not suggest moving the comment to its own line.

Lint is written in a way so that it expects either `///` documenation that documents macros or `/*` comment inside a huge macros. I.e. to silence linter you need to end line with macros with ` \` (or use NOLINT as usual). Here I would really prefer one `//` block which describes WTF is happening here (specifically why “for Windows” is guarded by `defined(_MAX_PATH) && !defined(MAXNAMLEN)` and not `defined(WIN32) && !defined(MAXNAMLEN)`) with a link to documentation or brief description of _MAX_PATH.

Though this comment would not be needed in first place if windows stuff was in os/win_defs.h like suggested.

>
> —
> Reply to this email directly or view it on GitHub.